Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -770,6 +770,18 @@ struct WebCore::ApplePayCouponCodeUpdate : WebCore::ApplePayDetailsUpdateBase {
 };
 #endif
 
+header: <WebCore/ScreenOrientationLockType.h>
+enum class WebCore::ScreenOrientationLockType : uint8_t {
+    Any
+    Landscape
```

Comment:
As mentioned in an earlier review, this should match the ordering of the enum as far as I know. Otherwise I wouldn't be surprised if encoding / decoding of the enum was completely wrong.

Reply:
oops, sorry. I'd assumed the enum was also alphabetically ordered.

Reply:
The whole point of this is to serialize / deserialize for IPC. It likely gets send as an unsigned integer and thus ordering is important if you want the same enum value on the other side of the pipe.

Reply:
yeah, makes sense. I'll be sure to check these more carefully next time.